pkgs/by-name/gt/gtimelog/package.nix

```diff
@@ -4,14 +4,14 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "gtimelog";
-  version = "unstable-2023-10-05";
+  version = "0.12.0";
   format = "setuptools";
```

Comment:
While we are at it, can we change this to pyproject = true?

Reply:
Did the change, please take another look.